docs/getting_started/experimental/running_programs_using_decorators.ipynb

```diff
@@ -216,65 +216,43 @@
    "id": "ca90f9be-aac9-4ed1-8384-463698914994",
    "metadata": {},
    "source": [
-    "## Program with local modules/folders\n",
+    "## QiskitPattern with local modules/folders\n",
     "\n",
-    "Situation with local folders is a little bit trickier. In order to make local imports work in remote execution of a program we need to specify `working_dir` argument for `distribute_program` decorator. It will tell quantum serverless to ship all content of current folder to remote cluster, which will make local folders discoverable by Python interpreter during remote execution.\n",
+    "Situation with local folders is a little bit trickier. In order to make local imports work in remote execution of a pattern we need to specify `working_dir` argument for `distribute_qiskit_pattern` decorator. It will tell quantum serverless to ship all content of current folder to remote cluster, which will make local folders discoverable by Python interpreter during remote execution.\n",
     "\n",
     "In this example we will use local folder `source_files` with `circuit_utils.py` file, which has implementation of `create_hello_world_circuit` function. "
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
+   "execution_count": null,
    "id": "8acd98bc-3443-447a-b8a1-c5e34a292f3b",
    "metadata": {},
-   "outputs": [
-    {
-     "data": {
-      "text/plain": [
-       "<Job | fa6431c1-3ee9-4989-a6de-63675e35a98b>"
-      ]
-     },
-     "execution_count": 6,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
+   "outputs": [],
    "source": [
     "import os\n",
-    "from quantum_serverless import distribute_program, distribute_task, get, save_result\n",
+    "from quantum_serverless import distribute_qiskit_pattern, distribute_task, get, save_result\n",
     "from qiskit.primitives import Sampler\n",
     "\n",
     "from source_files.circuit_utils import create_hello_world_circuit\n",
     "\n",
     "\n",
-    "@distribute_program(provider, working_dir=\"./\")\n",
-    "def my_program_with_modules():\n",
+    "@distribute_qiskit_pattern(provider, working_dir=\"./\")\n",
+    "def my_pattern_with_modules():\n",
     "    quasi_dists = Sampler().run(create_hello_world_circuit()).result().quasi_dists\n",
     "    return {\"quasi_dists\": quasi_dists}\n",
     "\n",
     "\n",
-    "job = my_program_with_modules()\n",
+    "job = my_pattern_with_modules()\n",
     "job"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 7,
+   "execution_count": null,
```

Comment:
do you want to execute this cell? (think there might also one other in this notebook that wasn't executed...)